tests/testit/test-list.R

```diff
@@ -0,0 +1,16 @@
+library(testit)
+
+assert('list_rmds() ignores thing in renv / packrat folder', {
+
+  dir = tempfile('testit-dir-')
+  dir.create(dir, recursive = TRUE)
+
+  dir.create(file.path(dir, 'renv'))
+  file.create(file.path(dir, 'renv/ignore.Rmd'))
+
+  rmd = blogdown:::list_rmds(dir)
+  (rmd %==% character())
+
+  unlink(dir, recursive = TRUE)
```

Comment:
FWIW I tried to use `on.exit()` here but it seems like the associated expression was being executed immediately, so I just clean up at the end instead.

Reply:
That's okay.